lib/internal/Magento/Framework/GraphQl/Query/QueryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\GraphQl\Query;
+
+/**
+ * Interface QueryInterface
+ *
+ * Contains information about GraphQl query
+ */
+interface QueryInterface
```

Comment:
Please finish description this and the other of Interface (including parameters and return values)